src/frame/sdk/DocumentAdvancedApi.ts

```diff
@@ -0,0 +1,32 @@
+import * as DocumentOperations from "./DocumentOperations";
+import ApiState from "../ApiState";
+import Future from "futurejs";
+import {documentToByteParts} from "../FrameUtils";
+import SDKError from "../../lib/SDKError";
+import {ErrorCodes} from "../../Constants";
+import EncryptedDekEndpoints from "../endpoints/EncryptedDekEndpoints";
+
+interface DecryptedUnmanagedDocumentResponse {
+    data: Uint8Array;
+}
+/**
+ * Retrieves sharing/meta information for the provided document ID. Then decrypts the provided data using the current users key and returns formatted doc structure.
+ * @param {string}     documentID        Unique lookup key of document to decrypt
+ * @param {Uint8Array} encryptedDocument Data of document to decrypt
+ */
+export function decryptLocalDoc(edeks: Uint8Array, encryptedDocument: Uint8Array): Future<SDKError, DecryptedUnmanagedDocumentResponse> {
```

Comment:
Pretty minor, but I'd maybe change this to be called `decryptWithProvidedEdeks` or `decryptWithEdeks` as I think that's much clearer than its current name.